wscript

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python
 # encoding: utf-8
+from gettext import install
```

Comment:
Why do we need this? I do not see this anywhere being used..

Reply:
I have no idea where this comes from!

Reply:
Please remove it.. :)